src/Express.js

```diff
@@ -1,75 +1,82 @@
 'use strict';
 
+import {Router} from 'ferry';
+
 import express from 'express';
 
-class Express {
-  constructor(specification, database) {
+class ExpressAdapter extends Router {
+
+  constructor(config = {}) {
+    super(config);
     this.name = 'Express';
-    this.specification = specification;
     this.app = express();
-
-    this.initialize();
   }
 
-  route(type, model) {
+  route(action, resource) {
 
-    switch (type) {
+    switch (action) {
 
       case 'index':
         return (req, res)=> {
-          res.send('Index');
+          res.send(action);
```

Comment:
Well, we wouldn't pull the models out, we would just use Ferry.storage directly. The storage instance itself would have handlers for each action required (the router has no knowledge of the models).

What should actually happen however, is that ferry and/or the storage engine should inject middleware to handle the action. That's what I'm hoping we can get working cleanly...